layouts/data-playground/list.html

```diff
@@ -31,13 +31,13 @@ <h2 class="text-2xl md:text-4xl font-bold mb-6">Welcome to our Data Playground!<
     <div id="all-license" class="mt-4">
         {{ partial "playground/all.html" . }}
     </div>
-    <div id="free-license" class="hidden">
+    <div id="free-license" class="hidden mt-4">
```

Comment:
I noticed that the cards appeared to "jump" when choosing the different filter options. Setting the margin on each div resolves that.